articles/security-center/tutorial-security-incident.md

```diff
@@ -17,7 +17,7 @@ ms.author: memildin
 ---
 
 # Tutorial: Triage, investigate, and respond to security alerts
-Security Center continuously analyzes your hybrid cloud workloads using advanced analytics and threat intelligence to alert you to malicious activity. You can also integrate alerts from other security products and services into Security Center, and create custom alerts based on your own indicators or intelligence sources. Once an alert is generated, swift action is needed to investigate and remediate. 
+Security Center continuously analyzes your hybrid cloud workloads using advanced analytics and threat intelligence to alert you on potential malicious activities in your cloud resources. Once an alert is raised, swift action is needed to investigate and remediate the potential security issue. 
```

Comment:
1. Please change "to alert you on potential malicious" to "to alert you about potentially malicious".

1. Why remove "You can also integrate alerts from other security products and services into Security Center." I think that sentence provides additional, useful information.